spark-jobs/src/main/scala/za/co/absa/enceladus/common/RecordIdGeneration.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.common
+
+import com.typesafe.config.{Config, ConfigException}
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.functions.callUDF
+import org.slf4j.{Logger, LoggerFactory}
+import za.co.absa.enceladus.utils.udf.{UDFLibrary, UDFNames}
+
+object RecordIdGeneration {
+
+  sealed trait UuidType
+
+  object UuidType {
+    case object TrueUuids extends UuidType
+    case object PseudoUuids extends UuidType
+    case object NoUuids  extends UuidType
+  }
+
+  private val log: Logger = LoggerFactory.getLogger(this.getClass)
+
+  def getRecordIdGenerationStrategyFromConfig(conf: Config): UuidType = {
+    val strategyValue = conf.getString("enceladus.recordid.generation.strategy")
+
+    strategyValue.toLowerCase match {
+      case "true" => UuidType.TrueUuids
```

Comment:
Since I have changed the pseudo variant, the `seededuiid` does not make sense anymore.

However, I was trying to avoid naming the pseudo variant anything with `uuid`, because the result does not guarantee uniqueness as the 'uUid' would suggest.

So maybe `uuid`, `hashid`, (`monotonousid` if used), and `none`?

Reply:
Of course my values were just suggestions. 😉 I see the issues with `seededuuid`
Does `hashId` imply that the value is stable tho?

Reply:
If one realizes what the hash is computed from, then it might. Maybe `stableHashId` then 😄

Reply:
I think it's good enough. Especially if described in documentation 😉

Reply:
I have updated the `Readme.md` with the options and a short description `foreach` 😄 .

Reply:
Sorry if being a nuisance. 😊  I don't think this is a description belongs to `README.md`.
Could you start a chapter in documentation Usage with `application.conf` parameters? Of course with this one there only for now, not all the others.

Reply:
OK, I have removed the added documentation from the `README.md` and I will create another PR to `gh-pages` instead.